docs/content/migration/v2.md

```diff
@@ -533,3 +533,27 @@ In `v2.11`, the `IPWhiteList` middleware is deprecated, please use the [IPAllowL
 ### IPWhiteList (TCP)
 
 In `v2.11`, the `IPWhiteList` middleware is deprecated, please use the [IPAllowList](../middlewares/tcp/ipallowlist.md) middleware instead.
+
+### TLS CipherSuites
+
+> By default, cipher suites without ECDHE support are no longer offered by either clients or servers during pre-TLS 1.3 handshakes.
```

Comment:
We should give more explanations on why we do this.

Reply:
This is a quote from the Go release.

Maybe you suggest improving the sentence bellow this quote?